### ISSUE TITLE
No database entry: Radio Thermostat Company of America [ID:c,Type:6501]

### ISSUE THREAD

**Issue by bigbasec** [cdjackson/HABmin#198]
Brand new CT30 that apparently has an issue.  Any chance it can get added?

**Comment by bigbasec** [comment]
Sorry, this is a CT101, not a CT30.  I am going to attempt to copy the 100 model and add a new one now, I'll keep you posted if it works!

**Comment by cdjackson** [comment]
Oh - just let me know if you want me to do anything. If it works ok, then create a pull request against the main openhab repository and I’ll take a look…

Cheers
Chris

**Comment by bigbasec** [comment]
Roger that... just learning a bit here, checked out the openHAB git, made my changes and FINALLY got it to build.  Now running it on the Pi and it seems as though it's doing a pretty large heal.  CPU load is up around 99% for about 1/2 hour.  Things seem VERY slow, can't even get Habmin to load(requests time out).  

So I'm going to let this ride for an hour or so and see what happens.  I'm not sure if it's how I built it or what.  I can only seem to start it using start_debug.sh, start.sh seems to hang after mDNS, not sure what that's about.  

I'll keep you posted if this ever finishes.  Thanks!

**Comment by bigbasec** [comment]
Looks like this is going to need to be changed.  The 101 is slightly different from the 100, as shown here : 
https://code.google.com/p/open-zwave/issues/detail?id=415&sort=-modified&colspec=ID%20Type%20Status%20Priority%20Milestone%20Owner%20Summary%20Modified

**Comment by bigbasec** [comment]
Any chance I can get the following added : 
CT101 : is a copy of the CT100, with type="6501" id="000c"

Also I have a couple little door sensors(made by everspring), and they report differently.  I've added this to openzwave and it seems to work just fine.  I wanted to test in openhab, but I can't seem to get a built version to run correctly, and if I just change the jar it's as if it never even sees it.  
<Product type="0001" id="0003" name="HSP02 Motion Detector"/>
<Product type="0002" id="0002" name="HSP02 Motion Detector"/>

Looks like they are just changing their ID's.

**Comment by cdjackson** [comment]
CT101 added with new codes. Note that we don't (currently) support the base=0 option that you refer to - I don't know how big a problem this is, but the database is being expanded so this can be added in future.

For the motion sensors, are they the same as one of the other Everspring sensors? IF so, please let me know which one and I'll copy over...

**Comment by bigbasec** [comment]
They are.  I am sending you an email now w/ the xml for that HSP02.  Sorry for the delay in response, been a little busy lately.  I made the changes manually, but it's nice to know they are in there for updates later on.  Thank you!

**Comment by bigbasec** [comment]
Hey Chris,
These are the updates for the HSP02 by EverSpring.  I used their manual to
make sure it lined up with the other sensors, I think there are some
differences, because this one has a LUX sensor.  It works out of the box w/
no settings, but this will allow the settings to be available to HABmin.

Thank you,

**Comment by bigbasec** [comment]
And I need to do some more looking into the pull request thing.  I made
changes, committed them, and then I see nothing when I try to do a pull
request on the Git site, so I'm sure I'm not doing something right.  :/

Sorry for the delays, after fighting w/ my zWave network (and Aeon stick)
for a week and change I finally through in the towel and got a Sigma
stick.  MAJOR improvement in just about everything.  Although I liked that
little "button" on the Aeon, it would constantly lock up and caused my CT30
to start broadcasting on it's own(a known problem).  I first thought it was
the CT30, so I went and bought the new USNAP module w/ the updated
firmware, and that didn't work.  Finally gave in and bought the Sigma, and
it's night and day.

I'm going to be looking into making the same style changes you made for the
"keyfob" to allow it to trigger the controller instead of a device, but I
have a GE 45600.   So I'll keep you posted if I make any progress.

Thanks again!
